code/game/machinery/washing_machine.dm

```diff
@@ -44,7 +45,21 @@
 	for(var/obj/item/clothing/suit/hooded/ian_costume/IC in contents)
 		new /obj/item/weapon/reagent_containers/food/snacks/meat/slab/corgi(src)
 		qdel(IC)
-
+	
+	for(var/obj/item/weapon/paper/P in contents)
+		if(crayon)
+			var/dye_color
+			if(istype(crayon,/obj/item/toy/crayon))
+				var/obj/item/toy/crayon/CR = crayon
+				dye_color = CR.colourName
+			else if(istype(crayon,/obj/item/weapon/stamp))
+				var/obj/item/weapon/stamp/ST = crayon
+				dye_color = ST.item_color

```

Comment:
This is often not an actual color, stamps tend to have 'hos' or 'clown' as colors.